### PR TITLE
77 up cased

### PR DESCRIPTION
**Pull request by levBagryansky**
Solving #77 
Implemented `text.up-cased` by eo.

### CODE REVIEW

**Comment by levBagryansky** [comment]
@OlesiaSub please review

File: src/test/eo/org/eolang/txt/text-tests.eo [hunk 1]
```diff
@@ -83,15 +83,25 @@
     (text "hello 1234").low-cased
     $.equal-to "hello 1234"
 
-[] > text-upper-case
+[] > text-upper-case-1
   assert-that > @
-    (text "HeLlO wOrLd!").up-cased
-    $.equal-to "HELLO WORLD!"
+    (text "HeLlO !").up-cased
```
Comment (OlesiaSub) [inline src/test/eo/org/eolang/txt/text-tests.eo:88]: why did you remove "world" everywhere?

File: src/test/eo/org/eolang/txt/text-tests.eo [hunk 2]
```diff
@@ -83,15 +83,25 @@
...
+    $.equal-to "HELLO !"
 
 [] > text-upper-case-2
   assert-that > @
-    (text "hello 123").up-cased
-    $.equal-to "HELLO 123"
+    (text "hel 3").up-cased
```
Comment (OlesiaSub) [inline src/test/eo/org/eolang/txt/text-tests.eo:93]: same question here, why test content was changed?

Comment (levBagryansky) [reply src/test/eo/org/eolang/txt/text-tests.eo:93]: I did it to reduce time of execution.

Comment (OlesiaSub) [reply src/test/eo/org/eolang/txt/text-tests.eo:93]: removing two symbols reduces time of execution? doesn't it mean that the method is _really_ slow?

Comment (levBagryansky) [reply src/test/eo/org/eolang/txt/text-tests.eo:93]: @OlesiaSub yes it is slow but it's not about the algorithm. At the moment the language is slow and the same algoritm would be much faster on java.

Comment (levBagryansky) [reply src/test/eo/org/eolang/txt/text-tests.eo:93]: @OlesiaSub We can try to use `thread`s here to calculate parts of the string in parallel when the are released

Comment (levBagryansky) [reply src/test/eo/org/eolang/txt/text-tests.eo:93]: @OlesiaSub We also can try to use `heap` but it similar to usage of mutability

Comment (OlesiaSub) [reply src/test/eo/org/eolang/txt/text-tests.eo:93]: @levBagryansky I think it's ok if it is slow because of the language. You can keep in mind the idea of using threads though and maybe implement it some time later.

**Comment by OlesiaSub** [comment]
@yegor256 this can be merged

File: src/test/eo/org/eolang/txt/text-tests.eo (hunk 2 above)
Comment (levBagryansky) [reply src/test/eo/org/eolang/txt/text-tests.eo:93]: @OlesiaSub Please also check #89. There is implementation by `heap`.

**Comment by levBagryansky** [comment]
@yegor256 please check

**Comment by yegor256** [comment]
@rultor merge
